src/legacy/core_plugins/kibana/public/dashboard/dashboard_app_controller.tsx

```diff
@@ -143,6 +142,16 @@ export class DashboardAppController {
     }
     $scope.showSaveQuery = dashboardCapabilities.saveQuery as boolean;
 
+    $scope.getShouldShowEditHelp = () =>
```

Comment:
Why are those put on the scope? I don't find a place in angular where they are used

Reply:
good point. they were on the scope previously, so I just left them there, but it's true that there's no good reason why they should remain there.